### ISSUE TITLE
Time created is always same as time modified after conversion

### ISSUE THREAD

**Issue by bgwwg** [wormi4ok/evernote2md#40]
Time modified is converted without mistakes. So time created is not correctly done.

**Comment by wormi4ok** [comment]
Hi @bgwwg , which OS do you use? On Unix systems `evernote2md` tries to use `touch` utility to set correct timestamps. If it cannot find `touch` binary, only time modified will be updated and time created is up to the OS to handle. Could it be the case that you observe?

**Comment by bgwwg** [comment]
Thanks, @wormi4ok , my OS is macOS 10.11

I can give you an example. A note with time tags shown below ends up being converted to a markdown file with time created same as time modified. 

```xml
<created>20180710T155144Z</created><updated>20180711T005501Z</updated>
```

**Comment by wormi4ok** [comment]
Fixed in version [v0.15.0](https://github.com/wormi4ok/evernote2md/releases/tag/v0.15.0)!
